<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>190</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>332</width>
    <height>190</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>332</width>
    <height>190</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AES GPU Cryptography</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/img/PadLockSmall.png</normaloff>:/img/PadLockSmall.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>321</width>
      <height>166</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="encryptTab">
     <attribute name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/img/Lock-icon.png</normaloff>:/img/Lock-icon.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Encrypt</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>5</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Input File:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputEncryptPath">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>5</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="inputBtn_encrypt">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>5</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Output File:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="outputEncryptPath">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>30</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="outputBtn_encrypt">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="encrpytProgress">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>301</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="encryptBtn">
      <property name="geometry">
       <rect>
        <x>224</x>
        <y>90</y>
        <width>86</width>
        <height>46</height>
       </rect>
      </property>
      <property name="text">
       <string>Encrypt</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/img/Actions-document-encrypt-icon.png</normaloff>:/img/Actions-document-encrypt-icon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="stopEncryptBtn">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>90</y>
        <width>76</width>
        <height>46</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/img/Stop-Normal-Red-icon.png</normaloff>:/img/Stop-Normal-Red-icon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="decryptTab">
     <attribute name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/img/UnLock-icon.png</normaloff>:/img/UnLock-icon.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Decrypt</string>
     </attribute>
     <widget class="QPushButton" name="inputBtn_decrypt">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>5</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputDecryptPath">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>5</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="outputDecryptPath">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>30</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="decryptBtn">
      <property name="geometry">
       <rect>
        <x>224</x>
        <y>90</y>
        <width>86</width>
        <height>46</height>
       </rect>
      </property>
      <property name="text">
       <string>Decrypt</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/img/Actions-document-decrypt-icon.png</normaloff>:/img/Actions-document-decrypt-icon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QProgressBar" name="decryptProgress">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>301</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="outputBtn_decrypt">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Output File:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>5</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Input File:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stopDecryptBtn">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>90</y>
        <width>76</width>
        <height>46</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/img/Stop-Normal-Red-icon.png</normaloff>:/img/Stop-Normal-Red-icon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="KeyTab">
     <attribute name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/img/key-icon (1).png</normaloff>:/img/key-icon (1).png</iconset>
     </attribute>
     <attribute name="title">
      <string>Key</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>25</y>
        <width>306</width>
        <height>56</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>5</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Key:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="encryptBtn_2">
      <property name="geometry">
       <rect>
        <x>215</x>
        <y>85</y>
        <width>96</width>
        <height>46</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/img/Generate-keys-icon.png</normaloff>:/img/Generate-keys-icon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="statusTab">
     <attribute name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/img/chip-icon.png</normaloff>:/img/chip-icon.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Status</string>
     </attribute>
     <widget class="QTextEdit" name="statusTxtBox">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>5</y>
        <width>306</width>
        <height>126</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
